apps/prairielearn/src/pages/instructorAssessmentGroups/instructorAssessmentGroups.ejs

```diff
@@ -11,7 +11,15 @@
 <body>
   <script>
     $(function () {
-      $('[data-toggle="popover"]').popover({ sanitize: false })
+      $('[data-toggle="popover"]').popover({ sanitize: false });
+      // Prevent the dropdown menu from closing when the popover is opened.
+      $('.js-group-action[data-toggle="popover"]').on("click", (e) => {
+        e.stopPropagation();
+      });
+      // Hide all popovers when the dropdown menu is closed.
+      $('.js-group-action-dropdown').on('hide.bs.dropdown', () => {
+        $('.js-group-action[data-toggle="popover"]').popover("hide");
+      });
```

Comment:
These changes seem to have broken the "Add members" option. Clicking to focus in the UID input closes both the dropdown and the popover.

Reply:
Indeed. I have removed the second check in 4c8e77c9fec320280a6df812ab7c5d4f06265331. This still causes the dropdown menu to close when clicking inside the popover, but this behaviour was already happening in master. I added a comment about this as well.

Other review comments also addressed in the same commit.

Reply:
Adding the following snippet was sufficient to keep the dropdown open when clicking in the popover; I also added in your original bit to close the popover when the dropdown closes.

```js
      $('.js-group-action-dropdown').on('hide.bs.dropdown', (e) => {
        // If the click is inside a popover, don't hide the dropdown.
        if (e.clickEvent.target.closest('.popover')) {
          e.preventDefault();
          e.stopPropagation();
          return;
        }

        // Hide all popovers when the dropdown menu is closed.
        $('.js-group-action[data-toggle="popover"]').popover("hide");
      });
```

Reply:
This worked really well, it even keeps the dropdown open if I hit "Cancel" in the popover (which is interesting).